src/main/java/org/opentripplanner/routing/impl/ATLFareServiceImpl.java

```diff
@@ -0,0 +1,375 @@
+            return RideType.XPRESS;
+        }
+
+        // Default to MARTA
+        return RideType.MARTA;
+        // TODO: Finish
+    }
+
+    private static TransferMeta classifyTransfer(RideType toRideType, RideType fromRideType, Fare.FareType fareType) {
+        switch (toRideType) {
+            case STREETCAR:
+                return new TransferMeta(TransferType.NO_TRANSFER);
+            case COBB_LOCAL:
+                if(!isElectronicPayment(fareType)) {
+                    if (fromRideType == RideType.COBB_LOCAL) {
+                        return new TransferMeta(TransferType.FREE_TRANSFER, 180, 4);
+                    }
+                    return new TransferMeta(TransferType.END_TRANSFER);
+                }
+                switch(fromRideType) {
+                    case COBB_LOCAL:
+                    case COBB_EXPRESS:
+                    case MARTA:
+                        return new TransferMeta(TransferType.FREE_TRANSFER, 180, 4);
+                    default:
+                        return new TransferMeta(TransferType.END_TRANSFER);
+                }
+            case COBB_EXPRESS:
+                if(!isElectronicPayment(fareType)) return new TransferMeta(TransferType.END_TRANSFER);
+                switch(fromRideType) {
+                    case COBB_EXPRESS:
+                    case MARTA:
+                        return new TransferMeta(TransferType.FREE_TRANSFER, 180, 4);
+                    case COBB_LOCAL:
+                        return new TransferMeta(TransferType.TRANSFER_PAY_DIFFERENCE, 180, 4);
+                    default:
+                        return new TransferMeta(TransferType.NO_TRANSFER);
+                }
+            case MARTA:
+               if(!isElectronicPayment(fareType)) return new TransferMeta(TransferType.END_TRANSFER);
+               switch(fromRideType) {
+                   case MARTA:
+                   case XPRESS:
+                   case COBB_LOCAL:
+                   case COBB_EXPRESS:
+                   case GCT_EXPRESS_Z1:
+                   case GCT_EXPRESS_Z2:
+                   case GCT_LOCAL:
+                       return new TransferMeta(TransferType.FREE_TRANSFER, 180, 4);
+                   default:
+                       return new TransferMeta(TransferType.END_TRANSFER);
+               }
+            case XPRESS:
+                if(!isElectronicPayment(fareType)) return new TransferMeta(TransferType.END_TRANSFER);
+                switch(fromRideType) {
+                    case MARTA:
+                    case COBB_EXPRESS:
+                    case GCT_EXPRESS_Z1:
+                    case GCT_EXPRESS_Z2:
+                    case XPRESS:
+                        // Could there be a situation where MARTA to XPRESS transfer is not pay on exit?
+                        return new TransferMeta(TransferType.FREE_TRANSFER, 180, 4, 0, true);
```

Comment:
Well that's going to be tough. I don't think we have enough data in this function right now to make this determination. 
It seems like the best way to do this would be to look at the time of the start of the leg and set it to pay on exit only in the evening.